stack.yaml

```diff
@@ -30,4 +30,5 @@ packages:
 extra-deps:
   - hlint-1.9.41
   - haskell-src-exts-1.19.1
+  - QuickCheck-2.10.1
```

Comment:
Definitely not. There is a version of QuickCheck in the `lts`. We should be using that.